packages/components/src/TextField/TextField.tsx

```diff
@@ -0,0 +1,130 @@
+import React, { useId } from "react"
+import classnames from "classnames"
+import { FieldGroup } from "~components/FieldGroup"
+import { FieldMessage } from "~components/FieldMessage"
+import { Input, InputProps } from "~components/Input"
+import { Label } from "~components/Label"
+import { SuccessIcon } from ".."
+import styles from "./TextField.module.scss"
+
+type OmittedInputProps =
+  | "startIconAdornment"
+  | "endIconAdornment"
+  | "ariaDescribedBy"
+  | "ariaLabel"
+
+export interface TextFieldProps extends Omit<InputProps, OmittedInputProps> {
+  /**
+   * A short example of input text. For context or additional information use the `description` prop
+   */
+  labelText: string | React.ReactNode
```

Comment:
Nit pick but these typings of `string | React.ReactNode` can just be `React.ReactNode`

Reply:
True!
```suggestion
  labelText: React.ReactNode
```